worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/CreditStreamManager.java

```diff
@@ -175,11 +175,6 @@ public void recycleStream(long streamId) {
     startRecycleThread(); // lazy start thread
   }
 
-  @VisibleForTesting
-  public int numStreamStates() {
```

Comment:
@FMX, the `numStreamStates` method is same as `getStreamsCount`, therefore this could be removed and `CreditStreamManagerSuiteJ` could use `getStreamsCount`.